File: Day5/PrimeNumber.cpp
#include<iostream>
using namespace std;

int checkPrimeNumber(int num){
    int isPrime = 1;
    if(num == 0  || num  == 1){
        isPrime = 0;
    }else{
        for(int i = 2; i<= num / 2; i++){
            if(num %i == 0){
                isPrime = 0;
                break;
            }
        }
    }
    return isPrime;
}
int main(){
    int num;
    cout<<"Enter the number:";
    cin>>num;
    int ans = checkPrimeNumber(num);
    if(ans == 0){
        cout<<num<<" is not a prime number";
    }else{
        cout<<num<<" is a prime number";
    }
    return 0;
}

File: Day5/prime_number.py
""" Program to check number prime or not"""

def get_check_prime_number(num):

    """Initially consider number is prime"""

    is_prime = True
    if num in (0,1):
        is_prime = False
    else:
        for i in range(2,num,1):
            if num % i == 0:
                is_prime = False
                break
    return is_prime

number = int(input("Enter the Number:"))
ANS = get_check_prime_number(number)
if ANS:
    print(number , "is a prime number")
else:
    print(number , "is not a prime number")
